aspnetcore/blazor/hybrid/class-libraries.md

```diff
@@ -0,0 +1,514 @@
+
+Update the RCL's `_Imports.razor` file to include the typical namespaces that Blazor Server apps use. The following example is a good starting point, and additional namespaces can be added as needed:
+
+`_Imports.razor` of the RCL:
+
+```razor
+@using System.Net.Http
+@using Microsoft.AspNetCore.Authorization
+@using Microsoft.AspNetCore.Components.Forms
+@using Microsoft.AspNetCore.Components.Routing
+@using Microsoft.AspNetCore.Components.Web
+@using Microsoft.AspNetCore.Components.Web.Virtualization
+@using Microsoft.JSInterop
+@using {PACKAGE ID/ASSEMBLY}
+@using {PACKAGE ID/ASSEMBLY}.Shared
+```
+
+As noted earlier, the `{PACKAGE ID/ASSEMBLY}` placeholder is the RCL's [package ID](/nuget/create-packages/creating-a-package-msbuild#set-properties). The package ID defaults to the project's assembly name if `<PackageId>` isn't specified in the project file. For an RCL named `SharedLibrary`, import namespaces for `SharedLibrary` and `SharedLibrary.Shared`.
+
+In `_Imports.razor` of the RCL after the `System` and `Microsoft` namespaces:
+
+```razor
+...
+@using SharedLibrary
+@using SharedLibrary.Shared
+```
+
+Update or add the namespace of the RCL in the Blazor Server's `_Imports.razor` file:
+
+* If the Blazor Server app's `Shared` folder remains in the app because the project exclusively uses remaining components in the folder, then *add* the RCL's `Shared` folder namespace to the file, along with adding the RCL's namespace:
+
+  ```diff
+  + @using {RCL PACKAGE ID/ASSEMBLY}
+  + @using {RCL PACKAGE ID/ASSEMBLY}.Shared
+  ```
+
+* If the Blazor Server app's `Shared` folder is empty because all of the assets are moved to the RCL, *update* the `Shared` folder namespace to the RCL's `Shared` folder namespace, along with adding the RCL's namespace:
+
+  ```diff
+  - @using {BLAZOR SERVER ASSEMBLY}.Shared
+  + @using {RCL PACKAGE ID/ASSEMBLY}
+  + @using {RCL PACKAGE ID/ASSEMBLY}.Shared
+  ```
+
+For example:
+
+```razor
+...
+@using SharedLibrary
+@using SharedLibrary.Shared
+```
+
+Update or add the namespace of the RCL in the Blazor Hybrid's `_Imports.razor` file:
+
+* If the Blazor Hybrid app's `Shared` folder remains in the app because the project exclusively uses remaining components in the folder, then *add* the RCL's `Shared` folder namespace to the file, along with adding the RCL's namespace:
+
+  ```diff
+  + @using {RCL PACKAGE ID/ASSEMBLY}
+  + @using {RCL PACKAGE ID/ASSEMBLY}.Shared
+  ```
+
+* If the Blazor Hybrid app's `Shared` folder is empty because all of the assets are moved to the RCL, *update* the `Shared` folder namespace to the RCL's `Shared` folder namespace, along with adding the RCL's namespace:
+
+  ```diff
+  - @using {BLAZOR HYBRID ASSEMBLY}.Shared
+  + @using {RCL PACKAGE ID/ASSEMBLY}
+  + @using {RCL PACKAGE ID/ASSEMBLY}.Shared
+  ```
+
+For example:
+
+```razor
+...
+@using SharedLibrary
+@using SharedLibrary.Shared
+```
+
+Place additional shared code and static assets in the RCL:
```

Comment:
The content in this section about handling static web assets is good. It just needs a few tweaks to make it generic and not specific to the project templates.